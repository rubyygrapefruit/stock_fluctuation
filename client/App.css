:root {
  --primary-color: #21ce99;
  --primary-light: #e6f9f3;
  --primary-bg: #21ce99;
  --primary-bglight: #e6f9f3;
  --lightbg: #efefef;
}
* {
  font-family: DIN Pro, -apple-system, BlinkMacSystemFont, sans-serif;
  font-weight: 500;
}
/* Overall app theme */
.theme-closed-down > * {
  color: #fff !important;
  background: #1b1b1d !important;
}

.theme-open-up > * {
  color: #1b1b1d !important;
  background: #fff !important;
}
/* Footer styles 
============== */
.uk-tab {
  margin-top: 0px;
  border-bottom: 1px solid #e5e5e5;
}
.uk-tab-closed {
  margin-top: 0px;
  border-bottom: 1px solid #000 !important;
}
.uk-tab::before {
  border: none;
}
.uk-tab > li > a {
  cursor: pointer;
}
/* Styles for market is open or closed */
.market-open {
  color: #1b1b1d !important;
  background: #fff;
}
.market-closed {
  color: #fff !important;
  background: #1b1b1d;
}
.market-is-up {
  color: var(--primary-color);
}
.uk-active.market-is-up > a {
  color: var(--primary-color) !important;
  border-color: var(--primary-color);
}
.market-is-down {
  color: #f45531;
}

.uk-active.market-is-down > a {
  color: #f45531 !important;
  border-color: #f45531;
}
