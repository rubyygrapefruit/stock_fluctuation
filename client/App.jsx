/* global document */
import React from 'react';
import ReactDOM from 'react-dom';
import './App.css';

//Components
import Header from './components/header/Header.jsx';
import GraphHeaderContainer from './components/graphHeader/GraphHeaderContainer.jsx';
import Graph from './components/graph/Graph.jsx';

const App = () => (
  <div className="uk-container uk-container-small">
    <Header />
    <GraphHeaderContainer />
    <Graph />
    <div>time tabs</div>
  </div>
);

export default App;
