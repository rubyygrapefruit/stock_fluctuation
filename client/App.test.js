import React from 'react';
import { shallow } from 'enzyme';
import App from './App.jsx';

describe('App', () => {
  test('should render a <div />', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  test('should render 4 children elements', () => {
    const wrapper = shallow(<App />);
    const wrapperDiv = wrapper.find('div').first();
    expect(wrapperDiv.children().length).toEqual(4);
  });
});
