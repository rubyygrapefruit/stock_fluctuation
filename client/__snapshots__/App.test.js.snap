// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render a <div /> 1`] = `
<div
  className="uk-container uk-container-small"
>
  <Header />
  <GraphHeaderContainer />
  <Graph />
  <div>
    time tabs
  </div>
</div>
`;
